ot/gromov/_bregman.py

```diff
@@ -343,6 +343,282 @@ def entropic_gromov_wasserstein2(
         return logv['gw_dist']
 
 
+def entropic_BAPG_gromov_wasserstein(
```

Comment:
naming ok ?
This is a bregman alternated projected gradient descent using D_{KL} for prox, so it is theoretically supposed to converge to exact solutions (like the proximal point).